src/main/java/net/javadiscord/javabot/command/moderation/ModerateCommand.java

```diff
@@ -0,0 +1,47 @@
+package net.javadiscord.javabot.command.moderation;
+
+import net.dv8tion.jda.api.entities.ChannelType;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.requests.restaction.interactions.ReplyCallbackAction;
+import net.javadiscord.javabot.command.ResponseException;
+import net.javadiscord.javabot.command.Responses;
+import net.javadiscord.javabot.command.interfaces.ISlashCommand;
+
+/**
+ * Basic moderation command.
+ */
+public abstract class ModerateCommand implements ISlashCommand {
+	private boolean allowThreads = true;
+
+	@Override
+	public ReplyCallbackAction handleSlashCommandInteraction(SlashCommandInteractionEvent event) throws ResponseException {
+		if (event.getGuild() == null) {
+			return Responses.error(event, "Moderation commands can only be used inside guilds.");
+		}
```

Comment:
```suggestion
		if (event.getGuild() == null) {
			return Responses.error(event, "This command can only be used inside servers.");
		}
```

Reply:
I think we should still stick with how discord officially calls servers/guilds
Guild is just an internal term and may cause confusion

Reply:
I agree.